pkg/operator/k8sutil/k8sutil_test.go

```diff
@@ -26,7 +26,7 @@ import (
 func TestTruncateNodeName(t *testing.T) {
 	// An entry's key is the result. The first value in the []string is the format and the second is the nodeName
 	tests := map[string][]string{
-		"rook-ceph-osd-prepare-47d1eb58e063be11c11c995b74cc5fbe-config": { // 61 chars
+		"rook-ceph-osd-prepare-8a49e4ac7d82f94a97234829dbf768fe1a2e9f2453b52d3d7c7c41ecfa65e01d-config": { // 61 chars
```

Comment:
You forgot to update the string length in the comment here.

```suggestion
		"rook-ceph-osd-prepare-8a49e4ac7d82f94a97234829dbf768fe1a2e9f2453b52d3d7c7c41ecfa65e01d-config": { // 93 chars
```